internal/generator/convert/convertv1.tmpl

```diff
@@ -44,7 +44,7 @@ func {{ .GOName }}FromV1Data(jsonData []byte) (ret {{ .GOType }}, err error) {
 
 	{{- range $j, $conv := .Conversions }}
     result = gjson.GetBytes(jsonData, "data.{{ $conv.OriginalName }}")
-    if result.Exists() {
+    if result.Exists() && result.Value() != nil {
```

Comment:
Might be able to get away with just `result.Value() != nil `.

Reply:
This is what I was thinking